_python_workout_code/3.1.py

```diff
@@ -0,0 +1,6 @@
+def firstlast(value):
+    return value[slice(0, len(value), len(value)-1)]
```

Comment:
very nice that you have found the `slice` function, but it's much easier to do `value[0] + value[-1]`